core/modules/qdisp/Executive.cc

```diff
@@ -470,7 +463,6 @@ void Executive::_waitUntilEmpty() {
         count = _requesters.size();
         _reapRequesters(lock);
         if(count != lastCount) {
-            lastCount = count;
```

Comment:
So in my understanding lastCount is always equal to -1, and  _requesters.size() can-t be equal to -1, so the test is always true, isn't it?